src/main/java/com/shopping_app/shoppingApp/mapping/AddressMapper.java

```diff
@@ -0,0 +1,21 @@
+package com.shopping_app.shoppingApp.mapping;
+
+import com.shopping_app.shoppingApp.domain.Address;
+import com.shopping_app.shoppingApp.model.Request.AddressRequest;
+import com.shopping_app.shoppingApp.model.Response.AddressResponse;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
+import org.mapstruct.ReportingPolicy;
+
+@Mapper(componentModel = "spring", unmappedTargetPolicy = ReportingPolicy.WARN)
+public interface AddressMapper {
+
+    @Mapping(target = "user",ignore = true)
```

Comment:
```suggestion
    @Mapping(target = "user", ignore = true)
```
Please make sure to leave a space after a comma

Reply:
removed mapper functionality